src/components/Author/index.js

```diff
@@ -1,7 +1,7 @@
-const Author = (props) => {
-  return (
-    <h4>{props.authors}</h4>
-  )
-}
+import { StyledAuthor } from "./author.styles";
 
-export default Author;
\ No newline at end of file
+const Author = ({ authors }) => {
+  return <StyledAuthor>{authors}</StyledAuthor>;
```

Comment:
is authors a string or a react component?

Reply:
It is an array. I've modified this component because I needed to map them anyway in case there are multiple authors.